base/src/consensus_state.rs

```diff
@@ -14,7 +14,17 @@ use wire_type::WireType;
 #[derive(Clone, Serialize, Deserialize, Default, PartialEq, Debug, WireType)]
 #[serde(from = "<Self as WireType>::WireType")]
 #[serde(into = "<Self as WireType>::WireType")]
-pub struct VrfOutputTruncated(Vec<u8>);
+pub struct VrfOutputTruncated(pub Vec<u8>);
+
+impl VrfOutputTruncated {
```

Comment:
What's this about Base64, I thought that only Base58 was used in Mina. If both are used we should add a new trait `Base64Encodable` that mirrors `Base58Encodable` and implement that here

Reply:
fixed

Reply:
Nice job 💯